api.bs

```diff
@@ -767,8 +799,12 @@ To <dfn>do attribution and fill a histogram</dfn>, given
 
     1.  If |impressions| is not empty:
 
+        1.  Let |key| be a [=privacy budget key=] whose items are |epoch| and |topLevelSite|.
```

Comment:
nit: This may need to be a [tuple](https://infra.spec.whatwg.org/#tuples) in order to "typecheck" with respect to the "items are..." phrasing.

Reply:
done